steam/ext/dota2/client.py

```diff
@@ -0,0 +1,127 @@
+"""Licensed under The MIT License (MIT) - Copyright (c) 2020-present James H-B. See LICENSE"""
+
+from __future__ import annotations
+
+import asyncio
+from typing import Final, Literal
+
+from ..._const import timeout
+from ..._gc import Client as Client_
+from ...app import DOTA2
+from ...ext import commands
+from ...utils import MISSING
+from .protobufs.dota_gcmessages_client_watch import (
+    CMsgClientToGCFindTopSourceTVGames,
+    CMsgGCToClientFindTopSourceTVGamesResponse,
+)
+from .state import GCState  # noqa: TCH001
+
+__all__ = (
+    "Client",
+    "Bot",
+)
+
+
+class Client(Client_):
+    """Represents a client connection that connects to Steam. This class is used to interact with the Steam API, CMs
+    and the Dota 2 Game Coordinator.
+
+    :class:`Client` is a subclass of :class:`steam.Client`, so whatever you can do with :class:`steam.Client` you can
+    do with :class:`Client`.
+    """
+
+    _APP: Final = DOTA2
+    _state: GCState  # type: ignore  # PEP 705
+
+    async def fetch_top_source_tv_games(
```

Comment:
```suggestion
    async def top_source_tv_games(
```
This can drop fetch_ because there isn't a get equivalent